.buildkite/test-pipeline.yaml

```diff
@@ -207,6 +207,7 @@ steps:
     - VLLM_USE_V1=1 pytest -v -s v1/structured_output
     - VLLM_USE_V1=1 pytest -v -s v1/test_stats.py
     - VLLM_USE_V1=1 pytest -v -s v1/test_utils.py
+    - VLLM_USE_V1 pytest -v -x tests/v1/entrypoints
```

Comment:
Missing the `=1` so actually running V0?

Reply:
good catch!

The structured output tests set the env var for each test, so that file should always use V1 fwiw

Reply:
😮‍💨